README.md

```diff
@@ -24,9 +24,9 @@ monitor progress, and troubleshoot issues when needed.
 
 ## Getting started
 
-Please visit the Airflow Platform documentation (latest **stable** release) for help with [installing Airflow](https://airflow.incubator.apache.org/installation.html), getting a [quick start](https://airflow.incubator.apache.org/start.html), or a more complete [tutorial](https://airflow.incubator.apache.org/tutorial.html).
+Please visit the Airflow Platform documentation (latest **stable** release) for help with [installing Airflow](https://airflow.apache.org/installation.html), getting a [quick start](https://airflow.apache.org/start.html), or a more complete [tutorial](https://airflow.apache.org/tutorial.html).
 
-Documentation of GitHub master (latest development branch): [ReadTheDocs Documentation](https://airflow.readthedocs.io/en/latest/)
+Documentation of GitHub master (latest development branch): [Read the Docs Documentation](https://airflow.readthedocs.io/en/latest/)
```

Comment:
Unrelated to this PR (so sorry for hijacking) but I would be happier if the default version you get upon visiting https://airflow.readthedocs.io (i.e. from Google) is for the same version you get if you `pip install apache-airflow`.

Reply:
We should mention this distinction on the content on airflow.apache.org too, or use something like https://robpol86.github.io/sphinxcontrib-versioning/ to be able to host multiple versions on the a.a.o site too.

Again, unrelated to this PR.

Reply:
@ashb I have changed the default branch on readthedocs. So https://airflow.readthedocs.io/ will now point to https://airflow.readthedocs.io/en/stable/ which is the version that is on pip.